app/AdminHotel.php

```diff
@@ -3,14 +3,30 @@
 namespace HotelBooking;
 
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
 
 class AdminHotel extends Model
 {
     //
-    protected $table='admin_hotels';
-
-    protected $fillable = ['hotel_id', 'username', 'password', 'name', 'email', 'phone'];
-
-    protected $hidden = ['password', 'remember_token'];
+    use SoftDeletes;
+    protected $table = 'admin_hotels';
+    protected $dates = ['deleted_at'];
+    protected $fillable = [
+        'hotel_id',
+        'username',
+        'password',
+        'name',
+        'email',
+        'phone',
+        'deleted_at'
+        ];
+    protected $hidden = [
+        'password',
+        'remember_token'
+        ];
 
+    public function getHotel()
```

Comment:
`getHotel`  method name is not right, this function is relationship, so the method name should be `hotel`.